tests/crescent/hooks/test_hook_order.py

```diff
@@ -0,0 +1,105 @@
+from __future__ import annotations
+
+from types import MethodType
+from typing import TYPE_CHECKING, Any
+
+from crescent import Bot, Plugin, command, hook
+from crescent.commands.groups import Group
```

Comment:
this can be imported from `crescent` i think

Reply:
Yeah, vscode auto-import is mostly to blame here.